File: sentimental-mario-less/mario.py
# Import cs50 library
from cs50 import get_int

# Prompt user for height
while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break

# Print out the piramid

# Outer loop to print out a single hash
for i in range(height):

    # Inner loop 1 to print out blank spaces
    for k in reversed(range(height - i - 1)):
        print(" ", end="")

    # Inner loop 2 to print out increasing amount of hashes depending on height
    for j in range(i):
        print("#", end="")

    # Print out the single hash
    print("#")

File: mario-less/mario.c
#include <cs50.h>
#include <unistd.h>

int main(void)
{
    int height;
    int i;
    int j;
    int k;

    height = 0;
    while (height < 1 || height > 8)
        height = get_int("Height: ");
    i = 0;
    while (i < height)
    {
        j = i;
        while (j < height - 1)
        {
            write(1, " ", 1);
            j++;
        }
        k = 0;
        while (k < i)
        {
            write(1, "#", 1);
            k++;
        }
        write(1, "#\n", 2);
        i++;
    }
    return (0);
}
